tests/e2e/e2e_test.go

```diff
@@ -192,59 +192,137 @@ func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
 	s.Require().NoError(err)
 
 	var (
-		denom0                    string = "uion"
-		denom1                    string = "uosmo"
-		tickSpacing               uint64 = 1
-		precisionFactorAtPriceOne int64  = -1
-		freezeDuration                   = time.Duration(time.Second)
-		swapFee                          = "0.01"
+		denom0                    string  = "uion"
+		denom1                    string  = "uosmo"
+		tickSpacing               uint64  = 1
+		precisionFactorAtPriceOne int64   = -1
+		freezeDuration                    = time.Duration(time.Second)
+		swapFee                           = "0.01"
+		swapFeeDec                sdk.Dec = sdk.MustNewDecFromStr("0.01")
+
+		// Defaults
+		defaultFeePerTx = sdk.NewInt(1000)
 	)
 
-	// helpers
+	// Helpers
 	var (
+		// Get current (updated) pool
 		updatedPool = func(poolId uint64) types.ConcentratedPoolExtension {
 			concentratedPool, err := node.QueryConcentratedPool(poolId)
 			s.Require().NoError(err)
 			return concentratedPool
 		}
+
+		// Get balances for address
+		addrBalance = func(address string) sdk.Coins {
+			addrBalances, err := node.QueryBalances(address)
+			s.Require().NoError(err)
+			return addrBalances
+		}
+
+		// Assert balances that are not affected by swap:
+		// * same amount of `stake` in balancesBefore and balancesAfter
+		// * amount of `e2e-default-feetoken` dropped by 1000 (default amount for fee per tx)
+		// * depending on `assertUosmo` and `assertUion` parameters, check that those balances have also not been changed
+		assertDefaultBalances = func(balancesBefore, balancesAfter sdk.Coins, assertUosmo, assertUion bool) {
+			s.Require().True(balancesAfter.AmountOf("stake").Equal(balancesBefore.AmountOf("stake")))
+			s.Require().True(balancesAfter.AmountOf("e2e-default-feetoken").Equal(balancesBefore.AmountOf("e2e-default-feetoken").Sub(defaultFeePerTx)))
+			if assertUion {
+				s.Require().True(balancesAfter.AmountOf("uion").Equal(balancesBefore.AmountOf("uion")))
+			}
+			if assertUosmo {
+				s.Require().True(balancesAfter.AmountOf("uosmo").Equal(balancesBefore.AmountOf("uosmo")))
```

Comment:
nit: I would prefer a more verbose name like `assertUionBalanceIsConstant`, but just personal preference. Feel free to resolve with no further action.

Reply:
yeah, that will actually improve readability, thanks!

Reply:
addressed, thanks again!